@navikt/core/react/src/util/i18n/locales/nb.ts

```diff
@@ -1,13 +1,12 @@
+import { Locale, nb } from "date-fns/locale";
+
 interface TranslationMap {
-  [component: string]:
-    | Record<string, string>
-    | {
-        [subComponent: string]: Record<string, string>;
-      };
+  [component: string]: Record<string, string | Record<string, string> | Locale>;
 }
 
 export default {
   global: {
+    dateLocale: nb,
```

Comment:
What do you think about putting the Locale here?

Reply:
Not a huge fan of directly connecting it with date-fns here. But until we have an alternative API, i think its' fine. The future update to solve it will be rather small and most likely not breaking for any applications, so no need to overcomplicate the API now.